scripts/process.ts

```diff
@@ -162,10 +214,28 @@ async function main() {
         { BOOTLOADER_TYPE: 'playground_batch' }
     );
 
-    if(!existsSync(OUTPUT_DIR)) {
+    console.log('Preprocessing bootloader tests');
+    const bootloaderTests = await renderFile('bootloader/tests/bootloader/bootloader_test.yul', {});
+
+    let testMethods = extractFunctionNames(bootloaderTests)
```

Comment:
nit: using `const` by default is a bit more idiomatical
```suggestion
    const testMethods = extractFunctionNames(bootloaderTests)
```